frontend/packages/console-app/src/components/quick-starts/controller/QuickStartFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { QuickStartStatus } from '../utils/quick-start-types';
+import { Button } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+
+import './QuickStartFooter.scss';
+
+type QuickStartFooterProps = {
+  status: QuickStartStatus;
+  onNext: () => void;
+  onBack: () => void;
+};
+
+enum PrimaryButtonText {
+  START = 'Start Tour',
+  NEXT = 'Next',
+  CLOSE = 'Close',
+}
+
+const getPrimaryButtonText = (status: string): PrimaryButtonText => {
+  switch (status) {
+    case QuickStartStatus.COMPLETE:
+      return PrimaryButtonText.CLOSE;
+    case QuickStartStatus.IN_PROGRESS:
+      return PrimaryButtonText.NEXT;
+    case QuickStartStatus.NOT_STARTED:
+    default:
+      return PrimaryButtonText.START;
+  }
+};
+
+const QuickStartFooter: React.FC<QuickStartFooterProps> = ({ status, onNext, onBack }) => (
+  <div className="co-quick-start-footer">
+    <Button
+      style={{
+        marginRight: 'var(--pf-global--spacer--md)',
+      }}
+      type="submit"
+      variant="primary"
+      onClick={onNext}
+      isInline
+    >
+      {getPrimaryButtonText(status)}
+    </Button>
+    {status !== QuickStartStatus.NOT_STARTED && (
+      <Button
+        style={{
+          marginRight: 'var(--pf-global--spacer--md)',
+        }}
+        type="submit"
+        variant="secondary"
+        onClick={onBack}
+        isInline
+      >
+        Back
+      </Button>
+    )}
+    {status === QuickStartStatus.COMPLETE && (
```

Comment:
I think you might want to check to see if we are on this route... otherwise you end up with a link that does nothing:

![Screen Shot 2020-07-29 at 10 06 57 PM](https://user-images.githubusercontent.com/8126518/88872302-db975500-d1e7-11ea-9d30-507286461d0e.png)

Reply:
Done.